### ISSUE TITLE
Functions parameters not mapped to query parameters for GET

### ISSUE THREAD

**Issue by antonantonovich** [davidmoten/odata-client#439]
Function arguments missed when we try to execute it.

CollectionPageNonEntityRequest.forFunction takes parameters as expected but serializes and puts it as request content, not a query part.  And later content ignored as request method is GET.

```
public static <T> CollectionPageNonEntityRequest<T> forFunction(ContextPath contextPath,
            Class<T> returnClass, Map<String, TypedObject> parameters) {
        String json = contextPath.context().serializer().serialize(parameters);
        return new CollectionPageNonEntityRequest<T>( //
                contextPath, //
                returnClass, //
                HttpMethod.GET, //   <--- method GET. it is correct
                Optional.of(json), //  <--- parameters stored as content. it's not correct
                HttpURLConnection.HTTP_OK);
    }
    
    CollectionPage<T> get(RequestOptions options) {
        ContextPath cp = contextPath.addQueries(options.getQueries());
        final HttpResponse r;
        List<RequestHeader> h = RequestHelper.cleanAndSupplementRequestHeaders(options, "minimal",
                method != HttpMethod.GET);
        if (method == HttpMethod.GET) {
            r = cp.context().service().get(cp.toUrl(), h, options); // <--- content ignored as we have GET
        } else {
            r = cp.context().service().post(cp.toUrl(), h, content.get(), options);
        }
```

**Comment by davidmoten** [comment]
ah ok, I'll have a look. I might update the title of this to make it more specific.

**Comment by davidmoten** [comment]
Fixed by PR #443. Fix is in release [0.1.97](https://github.com/davidmoten/odata-client/releases/tag/0.1.97) and is on Maven Central now. Please give it a try.

**Comment by antonantonovich** [comment]
Hello David. Fix works as expected. Thank you
